app/gui2/vite.config.ts

```diff
@@ -70,7 +70,6 @@ export default defineConfig({
       output: {
         manualChunks: {
           fontawesome: ['@fortawesome/react-fontawesome', '@fortawesome/free-brands-svg-icons'],
-          'aws-amplify': ['@aws-amplify/core', '@aws-amplify/auth'],
```

Comment:
I suppose the line was added for a reason here, but now it doesn't make sense anymore and breaks the build

Reply:
@Frizi maybe you can provide some context on why the line was added [here](https://github.com/enso-org/enso/pull/7873/commits/910133d42873f79742a9673d0daea5d9fad524b9#diff-b2a68bf45f9d84cea484d280b3ff4efee6b46143f6ffd30d75a169edbd1f00adR46)?